File: CS/ExportReportDemoEF/ExportReportDemoEF.Module/BusinessObjects/ExportReportDemoEFDbContext.cs
using System;
using System.Data;
using System.Linq;
using System.Data.Entity;
using System.Data.Common;
using System.Data.Entity.Core.Objects;
using System.Data.Entity.Infrastructure;
using System.ComponentModel;
using DevExpress.ExpressApp.EF.Updating;
using DevExpress.Persistent.BaseImpl.EF;

namespace  ExportReportDemoEF.Module.BusinessObjects {
	public class ExportReportDemoEFDbContext : DbContext {
		public ExportReportDemoEFDbContext(String connectionString)
			: base(connectionString) {
		}
		public ExportReportDemoEFDbContext(DbConnection connection)
			: base(connection, false) {
		}
		public DbSet<ModuleInfo> ModulesInfo { get; set; }
		public DbSet<ReportDataV2> ReportDataV2 { get; set; }
        public DbSet<Employee> Employees { get; set; }
    }
}

File: VB/ExportReportDemoEF/ExportReportDemoEF.Module/BusinessObjects/ExportReportDemoEFDbContext.vb
Imports System.Data.Entity
Imports System.Data.Common
Imports System.Data.Entity.Core.Objects
Imports System.Data.Entity.Infrastructure
Imports System.ComponentModel
Imports DevExpress.ExpressApp.EF.Updating
Imports DevExpress.Persistent.BaseImpl.EF

Namespace ExportReportDemoEF.Module.BusinessObjects

    Public Class ExportReportDemoEFDbContext
        Inherits DbContext

        Public Sub New(ByVal connectionString As String)
            MyBase.New(connectionString)
        End Sub

        Public Sub New(ByVal connection As DbConnection)
            MyBase.New(connection, False)
        End Sub

        Public Property ModulesInfo As DbSet(Of ModuleInfo)

        Public Property ReportDataV2 As DbSet(Of ReportDataV2)

        Public Property Employees As DbSet(Of Employee)
    End Class
End Namespace
